README.md

```diff
@@ -174,7 +174,7 @@ If you are interested in paid support, feel free to say so on the [jna-users mai
 License
 =======
 
-This library is licensed under the LGPL, version 2.1 or later, and (from version 4.0 onward) the Apache Software License, version 2.0. Commercial license arrangements are negotiable.
+This library is licensed under the LGPL, version 2.1 or later, or (from version 4.0 onward) the Apache License, version 2.0. Commercial license arrangements are negotiable. See [LICENSE](LICENSE) for more information.
```

Comment:
I would rephrase the "Commercial license arrangements" to "Commercial support" - as said above I don't see an option for a separate license.

Reply:
Agreed and done.